.history/frontend/src/App_20230118204115.js

```diff
@@ -0,0 +1,135 @@
+import { QueryClient, QueryClientProvider } from "react-query";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
```

Comment:
What are these .history files added for?

Reply:
Fixed .history issues. Added .history to .gitignore file and removed .history from the files for the pull request.